src/Kros.AspNetCore/Extensions/CorsExtensions.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Kros.AspNetCore.Extensions
+{
+    /// <summary>
+    /// Default Cors configuration helpers.
+    /// </summary>
+    public static class CorsExtensions
+    {
+        private const string AllowAllOriginsPolicy = "AllowAllOrigins";
+
+        /// <summary>
+        /// Adds allow all origins Cors policy.
+        /// </summary>
+        /// <param name="services">IoC container.</param>
+        public static IServiceCollection AddAllowAllOriginsCorsPolicy(this IServiceCollection services)
```

Comment:
We want to call it `AddAllowAnyOriginCors()`. No `All`.
Rename it in constants.